docs/index.md

```diff
@@ -1,4 +1,164 @@
 HCA DCP Query Service
 =====================
 
-[User Stories](USER_STORIES.md)
+# Introduction
+
+The HCA DCP Query Service provides an interface for scientists and developers to query metadata associated with
+experimental and analysis data stored in the [Human Cell Atlas](https://staging.data.humancellatlas.org/)
+[Data Coordination Platform](https://www.humancellatlas.org/data-sharing) (DCP). Metadata from the
+[DCP Data Store](https://github.com/HumanCellAtlas/data-store) are indexed and stored in an
+[AWS Aurora](https://aws.amazon.com/rds/aurora/) [PostgreSQL](https://www.postgresql.org/) database.
+
+# Query via the API
+Queries to the database can be sent over HTTP through the Query Service API
+[Swagger Documentation](https://query.data.humancellatlas.org/v1/ui/#/)
+or via the [Query Builder](https://query.data.humancellatlas.org/)
+
+## Executing Queries
+To execute a query via the swagger interface 
+- Click on the green `/query` row to expand it
+- Click `Try it out`
+- Edit the request body by adding in your query and any parameters. 
+
+For example
+```
+{
+  "params": {"s": 0},
+  "query": "select * from files where size > %(s)s limit 10"
+}
```

Comment:
error when i try this in swagger API ```{
  "detail": {
    "pgcode": "42601"
  },
  "status": 400,
  "title": "ERROR:  syntax error at or near \"%\"\nLINE 1: select * from files where size > %(s)s limit 10\n                                         ^\n",
  "type": "about:blank"
}```

Reply:
This was caused by a stale deployment. Please try again.